server/command.go

```diff
@@ -10,7 +10,11 @@ import (
 )
 
 const COMMAND_HELP = `* |/welcomebot preview [team-name] [user-name]| - preview the welcome message for the given team name. The current user's username will be used to render the template.
-* |/welcomebot list| - list the teams for which welcome messages were defined`
+* |/welcomebot list| - list the teams for which welcome messages were defined
+* |/welcomebot set_channel_welcome [welcome-message]| - set the welcome message for the given channel. Direct channels are not supported.
+* |/welcomebot get_channel_welcome| - print the welcome message set for the given channel (if any)
+* |/welcomebot delete_channel_welcome| - delete the welcome message for the given channel (if any)
+`
 
 func getCommand() *model.Command {
 	return &model.Command{
```

Comment:
GitHub isn't allowing me to add a comment on a non-changed line, but can you change the `AutoCompleteDesc` to include `list, set_channel_welcome, get_channel_welcome, delete_channel_welcome`?